src/explorer/AzureAccountTreeItem.ts

```diff
@@ -77,13 +78,16 @@ export class AzureAccountTreeItem extends AzureAccountTreeItemBase {
 
         if (trialAppContext.expirationDate < Date.now()) {
             this.trialAppNode = undefined;
+            await commands.executeCommand('setContext', 'trialAppExpired', true);
+            await commands.executeCommand('setContext', 'hasTrialApp', true);
```

Comment:
Are these contexts persistent? Like after you close out the editor?

Reply:
No. However, I accounted for that by setting the values here in AzureAccountTreeItem.